distributed/comm/ucx.py

```diff
@@ -46,7 +46,10 @@ def init_once():
     try:
         import rmm
 
-        cuda_array = lambda n: rmm.device_array(n, dtype=np.uint8)
+        if hasattr(rmm, "DeviceBuffer"):
+            cuda_array = lambda n: rmm.DeviceBuffer(size=n)
+        else:  # pre-0.12.0
```

Comment:
Should be `pre-0.11.0`. Fixing in PR ( https://github.com/dask/distributed/pull/3299 ).